roles/commcare_analytics/defaults/main.yml

```diff
@@ -87,7 +87,7 @@ certbot_domains:
   - www.example.com
 
 # Database backup variables
-postgresql_backup_enabled: false
+postgresql_backup_enabled: "{{ enable_postgresql_backup }}"
 postgresql_backup_dir: "/var/tmp"
-postgresql_backup_db_list: "commcare_analytics superset"
+postgresql_backup_db_list: "superset superset_cchq_data superset_meta superset_oauth2"
```

Comment:
is superset_cchq_data the datasources synced from HQ?

Reply:
Yes

Reply:
If its just the datasources, can we skip backing this up? If it has other tables then it seems fine to back up.

This way we can keep the backup limited to dbs we would need in case of recovery. We can always sync back the datasets.

Reply:
Good idea! I had a look through the code and it seems like we use it for the custom roles as well, but since we're able to create that on the fly I think this DB don't have to be backed up.

Reply:
Brilliant!
This should make the backup much lighter.